### PR TITLE
Prepare 1.4.13 release

### PR DESCRIPTION
**Pull request by ozancaglayan**

### CODE REVIEW

**Comment by mjpost** [comment]
Do any of you have time to add #103 to 1.4.13? I can try to do this myself by tomorrow. It would be nice to include.

**Comment by martinpopel** [comment]
Unfortunately, I don't have time this week (leaving offline in few minutes).

**Comment by mjpost** [comment]
I'm doing it right now.

**Comment by ozancaglayan** [comment]
I was looking at it as well but I'll let you do this. There are some issues that I can't guess how to address.

**Comment by ozancaglayan** [comment]
My PyPI ID is `ozancaglayan` btw.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ Install the Python module (Python 3 only)
 
     pip3 install sacrebleu
 
+In order to install Japanese tokenizer support through `mecab-python3`, you need to run the
+following command instead, to perform a full installation with dependencies:
+
+    pip3 install sacrebleu[ja]
+
 Alternately, you can install from the source:
```
Comment (mjpost) [inline README.md:26]: Can you explain this? What does the `[ja]` signify? Is this a bash wildcard?

**Comment by mjpost** [comment]
@ozancaglayan, can you merge in #109?

(In the future, you may want to just create branches on this repo, so we could all easily update them)

**Comment by mjpost** [comment]
@ozancaglayan, I added you to pypi. There are instructions for upload at the top of `setup.py`. It's very simple.

Please take care to run full tests before uploading. I guess this is already done via travis.

File: README.md (hunk 1 above)
Comment (ozancaglayan) [reply README.md:26]: This seems to be a good explanation:
https://stackoverflow.com/a/46775606

**Comment by ozancaglayan** [comment]
Closing this as changes are merged with the other WMT20 PR.

**Pull request closed without merging**